index.html

```diff
@@ -37,7 +37,144 @@ <h3 id="foot">LET’S CONNECT</h3>
-</body>
-</html>
+    </div>
+      <!--Work section-->
+    <div class="Work section">
+      <section class="card">
+        <img src="images/first.png" alt="" aria-hidden="true">
+        <h2>Tonic</h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button type='button'>See Project</button>
+      </section>
+    
+      <section class="card">
+        <img src="images/second.png" alt="" aria-hidden="true">
+        <h2> Multi-Post <br> Stories </h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button type='button'>See Project</button>
+      </section>
+    
+      <section class="card">
+        <img src="images/thired.png" alt="">
+        <h2>Tonic</h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <div>
+          <button type='button'>See Project</button>
+        </div>
+      </section>
+    
+      <section class="card">
+        <img src="images/forth.png" alt="" aria-hidden="true">
+        <h2>Multi-Post <br> Stories</h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <div>
+          <button type='button'>See Project</button>
+        </div>
+      </section>
+    </div>
+    
+    <!--About me-->
+    <div>
+      <section class="about">
```

Comment:
- [ ] Great working on you about section, kindly Add some `padding` between the social icons and button so they align with the Figma Template.

- [ ] Kindly work on the button design to look like the Figma Template

## Figma Template
![pad-4](https://user-images.githubusercontent.com/48371156/165937922-825d0015-76a4-48a0-b4c9-9f9539f9d5da.png)

## Your Design
![pad-3](https://user-images.githubusercontent.com/48371156/165937927-4fde1fd7-1545-461d-9c8d-bd5f2f8fbe92.png)